src/world/area_kkj/kkj_19/kkj_19_5_baking.c

```diff
@@ -193,10 +193,55 @@ API_CALLABLE(N(AwaitPlayerPressATimer)) {
     return ApiStatus_BLOCK;
 }
 
+#if VERSION_PAL
+struct N(BakingIngredient) {
+    s32 itemID;
+    s32 nameID;
+};
+struct N(BakingIngredient) N(BakingIngredientsNames)[] = {
+    ITEM_BAKING_FLOUR,      MSG_Menus_BakingFlour,
```

Comment:
each of these lines needs an open / close paren around it (see the warning mentioned in the bowser slug comment)